### ISSUE TITLE
About trained model

### ISSUE THREAD

**Issue by SoyeonUm** [mostafaelhoushi/DeepShift#13]
Thank you for providing your code :) 
I tried to train your code because there is no trained models.
However the accuracy is not high as much as you reported.
I really want to inference about DeepShift.
Could you provide your files of trained models?
Thank you

**Comment by mostafaelhoushi** [comment]
Thank you Soyeon for trying our code! I am glad that you are interested.
It may take me time to upload the models. Can you show me the command you typed that lead to bad results?

Thanks

**Comment by SoyeonUm** [comment]
Thank you for your answer.
Train command as follow:
python cifar10.py --arch resnet18 --shift-depth 1000 --shift-type PS --opt radam --gpu 0

Thanks

**Comment by mostafaelhoushi** [comment]
I am sorry that you got a low accuracy when you ran the command. I am now running it on my side to check.
What accuracy did you get? Can you attach the `train_log.csv` file in `<DeepShift root dir>/pytorch/models/cifar10/resnet18/shift_ps_all_wb_5/`

**Comment by SoyeonUm** [comment]
I trained with epochs 100 like below command.
_python cifar10.py --arch resnet18 --shift-depth 1000 --shift-type PS --opt radam --gpu 0 --epochs 100_
After training with 100 epochs, the train_top1_acc was reported as 98.296. However, when testing with trained model I made, the test top1 accuracy is only 9.92. I test it with the command below:
_CUDA_VISIBLE_DEVICES=0 python cifar10.py --arch resnet18 -e --weights models/cifar10/resnet18/shift_ps_all_wb_5_prev/checkpoint.pth.tar --shift-depth 1000 --use-kernel True --print-weights True_
Since the train accuracy is high, the train seems to have worked well, and I thought that the trained model was entered incorrectly, so I compared the weights when testing and it was the same. Did I use wrong test command ?
I attatched train_log.csv file in /pytorch/models/cifar10/resnet18/shift_ps_all_wb_5_prev/
[test_log.zip](https://github.com/mostafaelhoushi/DeepShift/files/6789768/test_log.zip)
Thank you :)

**Comment by mostafaelhoushi** [comment]
Can you try evaluating without `--use-kernel True`. It could be a bug in the CUDA kernels that need to be debugged.

**Comment by mostafaelhoushi** [comment]
During training, this is the trainig log I got. The validation accuracy exceeds 93%
[train_log.zip](https://github.com/mostafaelhoushi/DeepShift/files/6792272/train_log.zip)

**Comment by mostafaelhoushi** [comment]
And here are the pretrained weights: https://github.com/mostafaelhoushi/DeepShift/releases/download/v1.0/cifar10_resnet18_shift_ps_wb_5_from_scratch_checkpoint.pth.tar

**Comment by mostafaelhoushi** [comment]
I closed this issue but feel free to reopen it if you still have problems

**Comment by SoyeonUm** [comment]
Sorry for the late reply. Thank you for providing your pretrained weight and it works!

**Comment by SoyeonUm** [comment]
Could I ask for one more?
Could you provide trained model from pre-training model about Imagenet with Resnet50?
I really want to inference about that.
Thank you :)
